packages/medical-domain/src/domains/repositories/medical/datum.service.ts

```diff
@@ -100,9 +105,26 @@ const deduplicate = (data: Datum[], _opts: MedicalDataOptions): Datum[] => {
   })
 }
 
+export const filterOnDate = (data: Datum[], start: number, end: number, weekDaysFilter: WeekDaysFilter = defaultWeekDaysFilter): Datum[] => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏Unit test this?

Reply:
I think that this comment has been forgotten. Do you see value in unit testing this?

Reply:
Already done here 
packages/medical-domain/test/repositories/medical/datum.service.test.ts